README.rst

```diff
@@ -200,6 +203,47 @@ The template variables list:
 * ``${prenotazioni_folder_title}`` - Booking folder title.
 * ``${booking_requirements}`` - Related PrenotazioneType.booking_requirements field
 
+Note that the sms can be used only if you implement an own sender adapter
+Example:
+
+You just need to register a new adapter::
+
+    <adapter
+      factory = ".my_adapter.SMSSenderAdapter"
+    />
+
+And here the `send` method must bu implementend::
+
+    from logging import getLogger
+
+    from zope.component import adapter
+    from zope.component import getUtility
+    from zope.interface import Interface
+    from zope.interface import implementer
+
+    from redturtle.prenotazioni.content.prenotazione import IPrenotazione
+    from redturtle.prenotazioni.interfaces import IBookingNotificationSender
+    from redturtle.prenotazioni.interfaces import IBookingNotificatorSupervisorUtility
+    from redturtle.prenotazioni.interfaces import IPrenotazioneSMSMessage
+    from redturtle.prenotazioni.behaviors.notification_sms.adapter import BookingNotificationSender
+    logger = getLogger(__name__)
+
+
+    @implementer(IBookingNotificationSender)
+    @adapter(IPrenotazioneSMSMessage, IPrenotazione, YourAddonLayerInterface)
+    class CustomSMSSenderAdapter(BookingNotificationSender):
+        def __init__(self, message_adapter, booking, request) -> None:
+            self.message_adapter = message_adapter
+            self.booking = booking
+            self.request = request
+
+        def send(self):
+            if getUtility(
+                IBookingNotificatorSupervisorUtility,
+            ).is_sms_message_allowed(self.booking):
+                # Your custom send code here
+                pass
```

Comment:
se capisco bene, qui ci va qualcosa in più: tipo un esempio con i campi dei testi da inviare, il numero di celluare a cui inviare, ...


alla fine deve esserci scritto qualcosa del tipo

`custom_code_sms_send(phonenumber, message)`

e prima dove si recuperano il phonenumber e il message